config.default.yaml

```diff
@@ -145,16 +145,16 @@ electricity:
     H2: 168
 
   extendable_carriers:
-    Generator: [solar, onwind, offwind-ac, offwind-dc, OCGT]
+    Generator: [solar, onwind, offwind-ac, offwind-dc, OCGT, csp]
     StorageUnit: []  # battery, H2
-    Store: [battery, H2]
+    Store: [battery, H2, csp]
```

Comment:
My feeling is that it should be `csp -> architecture` flag which triggers adding stores for CSP. So, no need to specify `csp` here.